db/table_cache.cc

```diff
@@ -170,8 +170,9 @@ Status TableCache::FindTable(const ReadOptions& ro,
                            const_cast<bool*>(&no_io));
 
   if (*handle == nullptr) {
-    if (no_io) {  // Don't do IO and return a not-found status
-      return Status::Incomplete("Table not found in table_cache, no_io is set");
+    if (no_io) {
+      s = Status::Incomplete("Table not found in table_cache, no_io is set");
```

Comment:
Appveyor says:

```
    35>c:\projects\rocksdb\db\table_cache.cc(173): error C2065: 's': undeclared identifier [C:\projects\rocksdb\build\rocksdb.vcxproj]
    35>c:\projects\rocksdb\db\table_cache.cc(174): error C2065: 's': undeclared identifier [C:\projects\rocksdb\build\rocksdb.vcxproj]
```

I wonder why it thinks `s` is undeclared...

Reply:
It must have automatically rebased without any conflict and lost the declaration. I've never seen a clean rebase cause a problem before.